packages/component-library-react/src/Calendar/CalendarNavigationLabel.tsx

```diff
@@ -1,10 +1,9 @@
-import { FC } from 'react';
-import { Heading2 } from '../Heading2';
+import { FC, TimeHTMLAttributes } from 'react';
 
-interface CalendarNavigationLabelProps {
-  label?: string;
-}
+type CalendarNavigationLabelProps = TimeHTMLAttributes<HTMLTimeElement>;
 
-export const CalendarNavigationLabel: FC<CalendarNavigationLabelProps> = ({ label }) => (
-  <Heading2 className="utrecht-calendar__navigation-label">{label}</Heading2>
+export const CalendarNavigationLabel: FC<CalendarNavigationLabelProps> = ({ children, ...props }) => (
+  <time className="utrecht-calendar__navigation-label" {...props}>
+    {children}
+  </time>
```

Comment:
Taken from Den Haag datepicker, semantic element for time with machine readable time property.

Reply:
Nice I like `<time>` for this! However I think we might also want to include the `datetime` attribute then. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
It would mean adding an extra property though :)

Reply:
Oh never mind me, you've added that already! 👍😍